sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsWarning.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Text Analytics Warning.
+    /// </summary>
+    public struct TextAnalyticsWarning
+    {
+        internal TextAnalyticsWarning(string code, string message)
+        {
+            WarningCode = code;
+            Message = message;
+        }
+
+        /// <summary>
+        /// Code indicating the type of warning.
+        /// </summary>
+        public string WarningCode { get; }
```

Comment:
Was there going to be an extensible enum that went with this?

Reply:
Yes. will do later with issue #11826